salt/modules/restartcheck.py

```diff
@@ -555,6 +555,8 @@ def restartcheck(ignorelist=None, blacklist=None, excludepid=None, **kwargs):
             line = paths.stdout.readline()
```

Comment:
I see below things like: line = salt.utils.stringutils.to_unicode(line) (i.e. line 572). Should it do the same on line 555, i.e. line = salt.utils.stringutils.to_unicode(paths.stdout.readline()) ?

Reply:
Good catch. I missed that function. Will update the review.